SimpleLottery/Model/Lottery.swift

```diff
@@ -8,27 +8,34 @@
 
 import Foundation
 
+
 struct Lottery {
     
-    static let price = 1000
-    static let numbersRange = 1...45
-    static let numbersMaximumCount = 6
-    
-    let numbers: [Int]
+    // MARK - init
     
     init(numbers: [Int] = []) {
         self.numbers = Lottery.generatedNumbers(with: numbers)
     }
     
+    // MARK: - internal
+    
+    static let price = 1000
+    static let numberRange = 1...45
+    static let numberMaximumCount = 6
```

Comment:
```suggestion
    static let maximumCount = 6
```
사소한 부분이지만 count 와, number가 둘다 어떠한 숫자를 나타내는 단어인데 중복으로 쓰여서 변수명이 길어진듯 합니다. 하나만 쓰여도 의미가 통할 듯 합니다.
혹은 외부에서 쓰일 때 `maximum` 이라는 단어 때문에 뽑을 수 있는 숫자의 range가 있다고 느껴질 수 있습니다. `selectionNumber` 등이 괜찮지 않을까요? (좋은 단어가 생각안나네욤 -_)

Reply:
이미 오타가 들어가 있었네요.
원래 의도로는 멤버 변수 numbers의 maximum count를 뜻하는 numbersMaximumCount가 맞습니다.

해당 변수는 너무 길기도 하고 굳이 maximum이라고 특정 짓지 않고 numbers의 count라고만 전달해도 될 것 같아 numbersCount로 변수명을 바꾸어 주었습니다.